.taprc

```diff
@@ -1,5 +1,7 @@
 ts: false
 jsx: false
 flow: false
-check-coverage: false
-coverage: true
```

Comment:
as I wrote before, all this can be removed

Reply:
Apart from the the threshold configs the rest is removed now.